uikit/viewmodels/src/main/java/com/sygic/maps/uikit/viewmodels/searchresultlist/SearchResultListViewModel.kt

```diff
@@ -52,8 +53,10 @@ import com.sygic.sdk.search.SearchResult
  */
 @AutoFactory
 @Suppress("unused", "MemberVisibilityCanBePrivate")
-open class SearchResultListViewModel internal constructor(
-    private val searchManager: SearchManager
+open class SearchResultListViewModel @JvmOverloads internal constructor(
```

Comment:
Ten `searchResultListDataChangedObservable` tím pádem můžeš dát pryč. Nejde mi to napsat přímo k němu, tak to píši sem :)